extensions/parquet/table/src/test/java/io/deephaven/parquet/table/ParquetS3SimpleTestBase.java

```diff
@@ -0,0 +1,291 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.parquet.table;
+
+import io.deephaven.base.FileUtils;
+import io.deephaven.engine.table.ColumnDefinition;
+import io.deephaven.engine.table.Table;
+import io.deephaven.engine.table.TableDefinition;
+import io.deephaven.engine.table.impl.QueryTable;
+import io.deephaven.engine.testutil.junit4.EngineCleanup;
+import io.deephaven.engine.util.TableTools;
+import io.deephaven.extensions.s3.Credentials;
+import io.deephaven.extensions.s3.S3Instructions;
+import io.deephaven.extensions.s3.testlib.S3SeekableChannelTestSetup;
+import io.deephaven.extensions.s3.testlib.S3Helper;
+import io.deephaven.test.types.OutOfBandTest;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import software.amazon.awssdk.core.async.AsyncRequestBody;
+
+import java.io.File;
+import java.net.URI;
+import java.time.Duration;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import static io.deephaven.engine.testutil.TstUtils.assertTableEquals;
+import static io.deephaven.engine.util.TableTools.merge;
+import static io.deephaven.parquet.table.ParquetTools.writeKeyValuePartitionedTable;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+@Category(OutOfBandTest.class)
+abstract class ParquetS3SimpleTestBase extends S3SeekableChannelTestSetup {
```

Comment:
I called the other tests "simple" sort of to hint that they were only looking superficially at some bytes. I think this would be better named "S3ParquetTestBase". Similar w/ the derived classes.